.github/workflows/get-changed-files.yml

```diff
@@ -32,23 +32,23 @@ jobs:
           fetch-depth: 2
 
       - name: Check if component (`src` directory) files have changed
-        uses: tj-actions/changed-files@v35
+        uses: tj-actions/changed-files@v37
```

Comment:
Hi @adamdehaven, 🙌 author of [tj-actions/changed-files](https://github.com/tj-actions/changed-files) here. I just came across your workflow and couldn't help but notice a neat little tweak you could make. We've recently introduced a new `files_yaml` input which supports multiple filters in a single step, while still achieving the same results.

I think you might find it pretty handy! For more details, feel free to have a look at our [documentation](https://github.com/tj-actions/changed-files#using-local-git-history). 

Should you have any questions or need further assistance, please don't hesitate to ask. Have a fantastic rest of your day!